opentracing-api/pom.xml

```diff
@@ -29,8 +29,8 @@
 
     <properties>
         <main.basedir>${project.basedir}/..</main.basedir>
-        <main.java.version>1.6</main.java.version>
-        <main.signature.artifact>java16</main.signature.artifact>
+        <main.java.version>1.7</main.java.version>
```

Comment:
Though, it is "strange" (http://www.oracle.com/technetwork/java/eol-135779.html). Even JDK 7 is "end of public updates".